### PR TITLE
feat: add products screen

### PR DESCRIPTION
**Pull request by adith-a-danthi**
Added products screen.

https://deploy-preview-3--gametroid.netlify.app/pages/products.html

### CODE REVIEW

File: css/base.css [hunk 1]
```diff
@@ -29,7 +33,20 @@
   background-color: rgba(0, 0, 0, 0.5);
```
Comment (kedark152) [inline css/base.css]: use variables for colors

File: css/base.css [hunk 2]
```diff
@@ -29,7 +33,20 @@
...
 }
 
+.ecomm {
+  display: flex;
+  flex-direction: column;
+}
+
 .ecomm .card-img {
   height: 450px;
```
Comment (kedark152) [inline css/base.css]: try to use rems

File: css/products.css [hunk 3]
```diff
@@ -0,0 +1,27 @@
+.shop-layout {
+  display: grid;
+  grid-template-areas: "sidebar main-content";
+  grid-template-columns: 300px 1fr;
```
Comment (kedark152) [inline css/products.css]: try to use rems

**Review by kedark152 — COMMENTED** [review]
UI looks very cool & attractive. if time permits then try to make it responsive.
Also do some small changes suggested.
Use variables for colors as they may be used in future & use rems consistently.
also give some short description in PR.

File: pages/products.html [hunk 4]
```diff
@@ -17,20 +18,20 @@
...
-            <span class="fas fa-gamepad"></span>
+          <a href="/">
+            <span class="fas fa-gamepad text-primary"></span>
             gametroid
           </a>
         </h3>
         <ul class="nav-link-section">
-          <a href="" class="btn link">Login</a>
-          <a href="" class="btn fab small-fab btn-secondary">
+          <a href="./login.html" class="btn link">Login</a>
```
Comment (junaidshaikh-js) [inline pages/products.html]: `li` should be the direct child of `ul` 

Nest the `a` tag inside `li`

File: pages/products.html [hunk 5]
```diff
@@ -42,8 +43,300 @@ <h3 class="nav-title">
     </header>
 
     <!-- Content -->
-    <main class="container">
+    <main class="shop-layout">
+      <!-- Sidebar -->
+      <div class="sidebar">
```
Comment (junaidshaikh-js) [inline pages/products.html]: `aside` element can be used for side bars

File: pages/products.html [hunk 6]
```diff
@@ -42,8 +43,300 @@ <h3 class="nav-title">
...
+          </div>
+        </div>
+
+        <!-- Categories -->
+        <div class="my-4">
+          <h5 class="heading-5">Category</h5>
+          <ul class="sidebar-list">
+            <li>
+              <input class="input" type="checkbox" id="games" name="games" />
+              <label for="text-base"> Games </label>
```
Comment (junaidshaikh-js) [inline pages/products.html]: Looks like the `id` of input and `for` attribute of the label aren't matching and I noticed they aren't working in the preview.

File: pages/products.html [hunk 7]
```diff
@@ -42,8 +43,300 @@ <h3 class="nav-title">
...
+            </li>
+            <li>
+              <input
+                class="input"
+                type="checkbox"
+                id="consoles"
+                name="consoles"
+              />
+              <label for="text-base ml-2"> Consoles </label>
```
Comment (junaidshaikh-js) [inline pages/products.html]: This label is also not working.

File: pages/products.html [hunk 8]
```diff
@@ -42,8 +43,300 @@ <h3 class="nav-title">
...
+            </li>
+            <li>
+              <input
+                class="input"
+                type="checkbox"
+                id="accessories"
+                name="accessories"
+              />
+              <label for="text-base ml-2"> Accessories </label>
```
Comment (junaidshaikh-js) [inline pages/products.html]: This label too.

File: pages/products.html [hunk 9]
```diff
@@ -42,8 +43,300 @@ <h3 class="nav-title">
...
+          </ul>
+        </div>
+
+        <!-- Sort By -->
+        <div class="my-4">
+          <h5 class="heading-5">Sort By</h5>
+          <ul class="sidebar-list">
+            <li>
+              <input class="input" type="radio" id="price-low" name="sortBy" />
+              <label for="text-base"> Price (Low to High) </label>
```
Comment (junaidshaikh-js) [inline pages/products.html]: Sort by labels are also not working.

**Review by junaidshaikh-js — COMMENTED** [review]
Great work! 

Some suggestions: 
- Images of cards can be smaller, they are taking a lot of space 
- On a smaller screen, the layout can be changed into one column, filters followed by the cards. Later on, you can add a dropdown button for the filters.
- More space between the filter would make it look good.

**Review by junaidshaikh-js — APPROVED** [review]
Good to merge now!

**Pull request merged**